server/src/main/java/org/opensearch/action/search/AbstractSearchAsyncAction.java

```diff
@@ -451,7 +459,8 @@ private void onRequestEnd(SearchRequestContext searchRequestContext) {
     }
 
     private void executePhase(SearchPhase phase) {
-        try {
+        final Span phaseSpan = tracer.startSpan(SpanCreationContext.server().name("[phase/" + phase.getName() + "]"));
```

Comment:
We create new span once the phase starts execution (and scope the execution with it), however span is not closed by scope